hack/generate.sh

```diff
@@ -99,8 +99,8 @@ function generate_configuration() {
 
     sed -i "s|OS_IMAGES:.*|OS_IMAGES: '${OS_IMAGES}'|" ${__root}/deploy/podman/configmap.yml
```

Comment:
Consider creating `assisted-service/data/default_okd_release_images.json` and `assisted-service/data/default_okd_os_images.json`, so the source of truth can be entirely in `data/`

Reply:
Ah, good idea. Fixed in f277d54031fa945c805b0c7a5d8e1204b64b49de

Reply:
:ok_hand: